src/cmd/isolated_server.cpp

```diff
@@ -195,6 +202,10 @@ int main(int argc, const char* argv[]) {
       cout << "Server listening on " << port << endl;
     }
 
+    if (timeDelta > 0) {
```

Comment:
Don't need it here as called here https://github.com/Zilliqa/Zilliqa/blob/a5890ef13b5b0115912f4865d12a9f011732565e/src/libServer/IsolatedServer.cpp#L111-L140

Reply:
But then block will be incremented and generated even if timedelta = 0, Is this expected behavior?
I thought we wanted to have control over block generation for some special case.
If not, then can remove it.